ci/do_ci.sh

```diff
@@ -156,6 +156,7 @@ elif [[ "$1" == "format" ]]; then
   if [[ ! -z "$CHANGED" ]]; then
     echo "The following files have changes:"
     echo "$CHANGED"
+    git diff
```

Comment:
It's a pain when formatting tools fail. At least this should show what changed exactly.

Reply:
+1 to this!!!!

Reply:
@jsuereth - I decided to add this small patch in another simpler PR, so that it gets in sooner.

Reply:
@pyohannes - I updated the cmake formatting tools to latest, which matches latest as of now in Ubuntu 20.04. Test it all works in both 18.04 and 20.04, have not tested in other distros. Fixes #375